src/main/java/edu/uw/cse/testbayes/model/Bayes.java

```diff
@@ -1,28 +1,30 @@
 package edu.uw.cse.testbayes.model;
-
-import edu.uw.cse.testbayes.model.Probability;
-
 import java.lang.reflect.Method;
 import java.util.*;
 
+/*
+* This class calculates and maintains the probabilities on which the reordering of tests is based.
+* It has methods to return the next test to be run based on the pass or failure of other tests.
+*
+*/
 public class Bayes {
   // Map to store total probabilty
   private Map<String, Probability> tots;
 
   // Map to store Conditional probabilty of passing based on the fact that previous test passed
-  private Map<String, Map<String, Probability>> passconds;
+  private Map<String, Map<String, Probability>> passConds;
 
   // Map to store Conditional probabilty of passing based on the fact that previous test failed
-  private Map<String, Map<String, Probability>> failconds;
+  private Map<String, Map<String, Probability>> failConds;
 
   // Set to store already ran tests
   private Set<String> alreadyRan;
 
   //Numerator Parameter
-  public static final int NUMERATOR = 1;
+  private static final int NUMERATOR = 1;
 
   //denominator Parameter
-  public static final int DENOMINATOR = 2;
+  private static final int DENOMINATOR = 2;
 
   /*
    *@param testExecs : A map with all the execution strings mapping to a map that has test information like failing and passing times.
```

Comment:
The comments are inconsistent. Can you use a similar format to the other classes like `Probability.java`